app/controllers/api/physical_servers_controller.rb

```diff
@@ -64,16 +75,16 @@ def change_resource_state(state, type, id)
           server = resource_search(id, type, klass)
           desc = "Requested server state #{state} for #{server_ident(server)}"
           api_log_info(desc)
-          task_id = queue_object_action(server, desc, :method_name => state, :role => :ems_operations)
+          task_id = queue_object_action(server, desc, :method_name => state, :role => :ems_operations, :args => data)
           action_result(true, desc, :task_id => task_id)
-        rescue => err
+        rescue Exception => err
```

Comment:
if you do `rescue => err`, it should make code climate happy 😄

Reply:
But rescue => err, ruby assumes that you mean StandardError

Reply:
@igorrran that should be sufficient here, as far as I can tel